index.html

```diff
@@ -50,35 +50,79 @@ <h1>My Name</h1>
     <main tabindex="0">
       <section id="about">
         <header><h2>About Me</h2></header>
-        <p>
-          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nulla nisi
-          excepturi quidem, eum rem doloribus quam aspernatur hic enim eligendi
-          commodi minima ullam necessitatibus, cumque blanditiis, nihil magni
-          amet consectetur?
+        <p class="abt-me">
+          I'm Berwin Eibo, a dedicated software developer with a background in
+          education. My years of experience in teaching have honed my
+          communication skills and taught me the value of teamwork. I'm
+          passionate about creating innovative solutions to real-world problems
+          through code. I thrive on learning and eagerly embrace new challenges
+          in the dynamic world of technology.
         </p>
       </section>
+      <!-- projects -->
       <section id="projects">
         <header><h2>Projects Showcase</h2></header>
-        <p>
-          Lorem, ipsum dolor sit amet consectetur adipisicing elit. Nulla nisi
-          excepturi quidem, eum rem doloribus quam aspernatur hic enim eligendi
-          commodi minima ullam necessitatibus, cumque blanditiis, nihil magni
-          amet consectetur?
-        </p>
+        <div class="projects-container">
+          <div class="project1">
+            <p>
+              Web page developed to help refugees access bicycles, utilizing
+              HTML, CSS, and Flexboxes for layout.
+            </p>
+            <img src="bikes_for_refugees.jpg" alt="an image of a bike" />
```

Comment:
Excellent work on alt text but how could you improve this?

Why is this bike being shown? Do we already know it's an image or do we need to say? Google around this idea, of improving alt text, and think about what you might change.